test/fixtures/catalog/expect_check.yaml

```diff
@@ -6,6 +6,7 @@ description: |
 remediation: |
   ## Remediation
   Remediation text
+severity: critical
```

Comment:
Simple added to test the `severity: critical` scenario. I didn't want to create other extra test for this

Reply:
we need to improve these fixtures, maybe we could have some sort of generator/factory instead of relying on fixtures at all

Reply:
We could easily have a check factory. The fact is that I like to have the test fixtures, they are so easy to understand, and give a lot of more information to potential users on how things work. They could part of something else, rather than the evaluation unit tests.
Let's have this point on our radar!